scripts/ci/libraries/_local_mounts.sh

```diff
@@ -51,6 +51,7 @@ function local_mounts::generate_local_mounts_list {
         "$prefix"setup.py:/opt/airflow/setup.py:cached
         "$prefix"tests:/opt/airflow/tests:cached
         "$prefix"kubernetes_tests:/opt/airflow/kubernetes_tests:cached
+        "$prefix"chart_tests:/opt/airflow/chart_tests:cached
```

Comment:
I think you should also add /chart/ directory. See: https://github.com/apache/airflow/pull/11694/files